### PR TITLE
Update to 1.0.20 and use macros for versions

### LINKED ISSUE #16: Move version numbers into macros
The version numbers on the install page are hard coded right now. It would be easier to maintain if we could [define them as macros](https://squidfunk.github.io/mkdocs-material/reference/variables/).

### PR DESCRIPTION
**Pull request by rolandlo**
Closes #16

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -23,6 +23,15 @@ theme:
     primary: indigo
     accent: deep orange
 
+#Enable macros
+plugins:
+  - macros
```
Comment (Technius) [inline mkdocs.yml:28]: You need to explicitly add the search plugin again according to the [documenation](https://www.mkdocs.org/user-guide/configuration/#plugins).

**Comment by Technius** [comment]
We probably also need to update https://github.com/xournalpp/xournalpp.github.io/blob/5daf106ef9c46f6ee415234e9e1ab8ecd3e524c0/.github/workflows/deploy.yml#L16
to `1.16` and then make sure we have a `requirements.txt` with `mkdocs-macros-plugin` in it.

**Comment by rolandlo** [comment]
> to `1.16` and then make sure we have a `requirements.txt` with `mkdocs-macros-plugin` in it.

Is the `requirements.txt` just for the user or does it play a role when the site is deployed? How should it look like? I couldn't find the docs to show that.

**Comment by Technius** [comment]
The `requirements.txt` is one of the standard ways to specify dependencies in Python. The deploy plugin will use `pip` to install the packages in `requirements.txt`, so we can add extra plugins there.

**Comment by rolandlo** [comment]
I have added the requirements.txt file in the same folder as the `deploy.yml` file.

**Comment by Technius** [comment]
Ok, let's try merging this. If it doesn't work, the `requirements.txt` is probably in the wrong folder (could try the root of the repository instead).

**Comment by rolandlo** [comment]
> Ok, let's try merging this. If it doesn't work, the `requirements.txt` is probably in the wrong folder (could try the root of the repository instead).

You were right about this. Now it works with the requirements.txt file in the root folder.

**Pull request merged**